### PR TITLE
DB config added

### PR DESCRIPTION
**Pull request by JohnOldman2018**

### CODE REVIEW

**Review by amussagaliyev — COMMENTED** [review]
Let's use yaml format
Please change file's extension to yml and reformat content accordingly

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,4 +1,7 @@
-server.port = 3306
-spring.datasource.url=jdbc:h2:dev
-spring.datasource.username=sql11436834
-spring.datasource.password=2fr5rNKsI6
+spring:
+    datasource: \
```
Comment (amussagaliyev) [inline src/main/resources/application.properties]: What are these slashes? I think we don't need them

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+        server: sql11.freemysqlhosting.net
+        name:sql11436834
+        username: sql11436834
+        password: 2fr5rNKsI6\
```
Comment (amussagaliyev) [inline src/main/resources/application.properties]: Slash again

**Review by amussagaliyev — COMMENTED** [review]
These changes does not meet AC. Please check

**Review by JohnOldman2018 — COMMENTED** [review]
Strangly enough, there are no slashes in my files, even if I open them with Notepad++
Tests runs with errors.

**Review by JohnOldman2018 — COMMENTED** [review]
Вот такие ошибки:
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 7.838 s <<< FAILURE! - in com.jantoleu.rooms_manager.ApplicationTests
[ERROR] contextLoads  Time elapsed: 0.022 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: Invocation of init method failed; nested exception is java.lang.RuntimeException: Driver com.mysql.cj.jdbc.Driver claims to not accept jdbcUrl, sql11.freemysqlhosting.net
Caused by: java.lang.RuntimeException: Driver com.mysql.cj.jdbc.Driver claims to not accept jdbcUrl, sql11.freemysqlhosting.net
Не хватает библиотек Джава.

**Comment by JohnOldman2018** [comment]
Арман,
Не хватает бинов Джава, вроде.
Тесты не проходятся.

вт, 14 сент. 2021 г. в 15:38, amussagaliyev ***@***.***>:

**Comment by JohnOldman2018** [comment]
Как и требовалось, я обновил драйвер на com.mysql.cj.jdbc.Driver
Но он не может получить бины.

java.lang.IllegalStateException: Failed to load ApplicationContext

Caused by: org.springframework.beans.factory.BeanCreationException: Error
creating bean with name 'liquibase' defined in class path resource
[org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]:
Invocation of init method failed; nested exception is
java.lang.RuntimeException: Driver com.mysql.cj.jdbc.Driver claims to not
accept jdbcUrl, sql11.freemysqlhosting.net

Caused by: java.lang.RuntimeException: Driver com.mysql.cj.jdbc.Driver
claims to not accept jdbcUrl, sql11.freemysqlhosting.net

вт, 14 сент. 2021 г. в 15:38, amussagaliyev ***@***.***>:

**Review by JohnOldman2018 — COMMENTED** [review]
Changed db config file and commented out liquibase dependency.

**Pull request merged**